### PR TITLE
rspec 3

### PR DESCRIPTION
**Pull request by bensymonds**
Depends on https://github.com/withassociates/slices/pull/119.

### CODE REVIEW

**Comment by erino** [comment]
LGTM

**Comment by bensymonds** [comment]
Thanks @erino, but there are a few deprecation warnings I need to fix too, so will ask you to look at this again in a bit :smile:

**Comment by bensymonds** [comment]
This is ready to go now. The changes are very boring.

**Comment by jgwhite** [comment]
You’re making Sam Phippen very happy. LGTMMMMMMMMMM

File: spec/helpers/navigation_help_spec.rb [hunk 1]
```diff
@@ -28,11 +28,11 @@ def it_does_not_render_for(path, level, depth = 1)
...
   end
 
-  failure_message_for_should do |actual|
+  failure_message do |actual|
     "expected that\n#{actual}\nwould be equlivent to\n#{expected}"
   end
 
-  failure_message_for_should_not do |actual|
+  failure_message_when_negated do |actual|
     "expected that\n#{actual} would not be equlivent to\n#{expected}"
```
Comment (calumgunn) [inline spec/helpers/navigation_help_spec.rb]: That typo of `equivalent` is making my eye twitch

**Comment by calumgunn** [comment]
Looks good to me!

File: spec/helpers/navigation_help_spec.rb (hunk 1 above)
Comment (jgwhite) [inline spec/helpers/navigation_help_spec.rb]: Oh crikey yeah

Comment (bensymonds) [inline spec/helpers/navigation_help_spec.rb]: I was going to refuse to fix typos, but given this is open source I will. :smile:

Comment (bensymonds) [inline spec/helpers/navigation_help_spec.rb]: There you go.

Comment (calumgunn) [inline spec/helpers/navigation_help_spec.rb]: You're an angel!

Comment (bensymonds) [inline spec/helpers/navigation_help_spec.rb]: I try. Going to swap the commits around so it's clear I didn't introduce the typo in the first commit :sunglasses:

**Pull request merged**